### ISSUE TITLE
logically weighted index

### ISSUE THREAD

**Issue by guenp** [giulioungaretti/opendatahacklab2014#10]
e.g. age should be 1 when your age is near that value, instead of being 1 when largest.

**Comment by giulioungaretti** [comment]
yeah @hsuominen  and I were discussing aobut that asweel and indeed that's the best solution :+1:
